pkg/gds/emails/sentlogs.go

```diff
@@ -0,0 +1,37 @@
+// This contains helper functions to help confirm if emails were sent.
+package emails
```

Comment:
This actually looks like a good fit for the `models` package, may I suggest moving this into the `pkg/gds/models/v1/contacts.go` since it's a direct check against the `Contacts` object? Also, can we pass in the Contacts directly into the method instead of the VASP?